### PR TITLE
Tyr xenonauten now layers over the helmet so it displays as   intended

### PR DESCRIPTION
**Pull request by SplinterGP**
## Why It's Good For The Game
what title says, but by consequence also anything that layers under helmet :P

## Changelog
:cl:
expansion: helmets now layer under armor.
fix: Hlin now appears on xenonauten properly.
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
get kuro to approve this

**Review by Kuro020 — APPROVED** [review]

**Comment by SplinterGP** [comment]
why tf these files were changed, hell

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -397,14 +397,14 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
...
-#define GOGGLES_LAYER 11	//For putting Ballistic goggles and potentially other things above masks
-#define HEAD_LAYER 10
+#define HAIR_LAYER 17 //TODO: make part of head layer?
+#define EARS_LAYER 16
+#define FACEMASK_LAYER 15
+#define GOGGLES_LAYER 14	//For putting Ballistic goggles and potentially other things above masks
+#define HEAD_LAYER 13
+#define BACK_LAYER 12
+#define SUIT_LAYER 11 //Possible make this an overlay of somethign required to wear a belt?
+#define SUIT_STORE_LAYER 10
```
Comment (SplinterGP) [inline code/__DEFINES/mobs.dm:407]:
```suggestion
#define HAIR_LAYER 17 //TODO: make part of head layer?
#define EARS_LAYER 16
#define FACEMASK_LAYER 15
#define GOGGLES_LAYER 14	//For putting Ballistic goggles and potentially other things above masks
#define HEAD_LAYER 13
#define SUIT_LAYER 12 //Possible make this an overlay of somethign required to wear a belt?
#define SUIT_STORE_LAYER 11
#define BACK_LAYER 10
```

**Comment by lKiseki** [comment]
Just putting this here for reference.
beforeafter

I've already talked to Splinter in Discord about this, but it seems this PR screwed up layers in a lot of areas, as made evident by the above picture _(notice the bandanna, gun sling, and hair differences)_.

**Comment by Kuro020** [comment]
it would seem the best bet is making a new layer for this

**Comment by SplinterGP** [comment]
added a fix to hlin since it was only one line

**Review by BraveMole — APPROVED** [review]

**Pull request merged**